salt/modules/postgres.py

```diff
@@ -886,7 +886,7 @@ def get_bool(rowdict, key):
         for date_key in ('expiry time',):
             try:
                 retrow[date_key] = datetime.datetime.strptime(
-                    row['date_key'], '%Y-%m-%d %H:%M:%S')
+                    row[date_key], '%Y-%m-%d %H:%M:%S')
```

Comment:
Just to be clearer, the reason for that is that the query above doesn't select any `"date key"` column. Meaning that this used to invariably raise a `KeyError` that is handled below.

I'm unsure why `KeyError` would still need to be caught though. `strptime` can only raise `ValueError` and according to the query, I think `expiry time` key will always exist. Let me know if you think `KeyError` is superfluous.